irohad/consensus/yac/messages.hpp

```diff
@@ -34,17 +34,20 @@ namespace iroha {
         YacHash hash;
         model::Signature signature;
 
+          //TODO: I am not sure that rhs is a great variable name choice here. This is a comparison operator
```

Comment:
This is a general naming for implementing operators -- right-hand side. http://en.cppreference.com/w/cpp/language/operator_comparison